mcp_zammad/server.py

```
@@ -578,17 +590,29 @@
+                # Extract ticket_id and update fields separately
+                update_data = params.model_dump(exclude={"ticket_id"}, exclude_none=True)
+                ticket_data = client.update_ticket(ticket_id=params.ticket_id, **update_data)
+                return Ticket(**ticket_data)
+            except Exception as e:
+                error_msg = str(e).lower()
+                if "not found" in error_msg or "couldn't find" in error_msg:
+                    raise ValueError(
+                        f"Ticket ID {params.ticket_id} not found. "
+                        f"Note: Use the internal 'id' field from search results, not the display 'number'. "
+                        f"Example: For ticket #65003, search first to find its internal ID."
+                    ) from e
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)